data/inline-histogram.go

```diff
@@ -180,17 +181,20 @@ func (ih *InlineHistogram) String() string {
 	return s
 }
 
-func (ih *InlineHistogram) HistogramToEmpiricalString() string {
-	s := fmt.Sprintf("InlineHistogram:\nBinCount: %v\nObservations: %v\nMin: %f\nMax: %f\nMean: %f\n", len(ih.bins), ih.pm.sampleSize, ih.pm.min, ih.pm.max, ih.pm.GetMean())
-	s += "Bin Start"
-	for idx, _ := range ih.bins {
-		s += fmt.Sprintf("%f, ", ih.minValue+(ih.binWidth*float64(idx)))
+func (ih *InlineHistogram) Bins() []float64 {
+	binStarts := []float64{}
+	for i, _ := range ih.bins {
+		binStarts = append(binStarts, float64(i))
```

Comment:
shouldnt this start with  min bin value and increment by bin width?

Reply:
Is the index of ih.bins not the same as the bin starts (and thereby the ih.minValue+ increments of ih.binWidth)?

Reply:
the indexes are the same but to calculate bin starts you have to offset by the min value and multiply by bin width.

Reply:
line 187 should contain something like ih.minValue + (float64(i)*ih.binWidth)

Reply:
also, append is probably slow, you know the count based on the length of bins, so you can preallocate the array to the right size.